frontend/src/bundles/studio/components/video-menu/components/menu-body/menu-body.tsx

```diff
@@ -30,21 +29,35 @@ const MenuBody: React.FC<Properties> = ({
                         left: '100px',
                         height: '72vh',
                         width: '335px',
-                        p: 2,
+                        padding: '12px 18px 0 18px',
                         zIndex: 1000,
                         borderRadius: '8px',
                         boxShadow: 'lg',
                         bg: 'background.900',
                         color: 'white',
```

Comment:
better to move styles to module

Reply:
If I move these styles to css module and add className to the component, they will not be applied.
I guess it is overridden by some chakra styles.
sx prop works here because it has higher specificity than className and chakra styles